src/nvim/normal.c

```diff
@@ -3178,11 +3178,11 @@ static void nv_zet(cmdarg_T *cap)
       FOR_ALL_WINDOWS_IN_TAB(wp, curtab) {
         if (wp != curwin && foldmethodIsDiff(wp) && wp->w_p_scb) {
           wp->w_p_fen = curwin->w_p_fen;
-          changed_window_setting_win(wp);
+          foldChanged(wp);
         }
       }
     }
-    changed_window_setting();
+    foldChanged(curwin);
```

Comment:
If `FoldChanged` is triggered here, then it should also be triggered whenever 'foldenable' is changed.